src/applications/personalization/view-dependents/manage-dependents/containers/ManageDependentsApp.jsx

```diff
@@ -4,16 +4,22 @@ import LoadingIndicator from '@department-of-veterans-affairs/component-library/
 import PropTypes from 'prop-types';
 import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
 import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+import ErrorMessage from 'platform/forms/components/common/alerts/ErrorMessage';
 import * as actions from '../redux/actions';
 import { SCHEMAS } from '../schemas';
-import { submitToApi } from '../utils';
+import {
+  transformForSubmit,
+  ServerErrorFragment,
+  LOADING_STATUS,
```

Comment:
We should probably refactor the main view dependents redux code to use this as well at some point